recipes-bsp/firmware-pack/firmware-pack.bb

```diff
@@ -0,0 +1,51 @@
+SECTION = "bootloaders"
+SUMMARY = "Firmware Packaging"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/BSD-3-Clause;md5=550794465ba0ec5312d6919e203a55f9"
+
+inherit deploy
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+DEPENDS = " trusted-firmware-a u-boot bootparameter-native fiptool-native"
+
+S = "${WORKDIR}"
+
+do_configure[noexec] = "1"
+
+do_compile () {
```

Comment:
In task do_compile , I will explain the details:
 
- Create bl2_bp.bin by using write data to sector 1 and add file bl2-rzg2l-sbc.bin at the end of the file bl2_bp.bin
- Create fip.bin using fiptool 
- Convert file bl2_bp.bin to bl2_bp.srec,fip.bin to fip.srec.

- Note: I removed do_configure[noexec] = "1" from this recipe because do_configure will be not executed

Reply:
Updated in the commit: bdfa5bcd3f027ed07aa0aa9c6b8367eada43a1c4